tests/Microsoft.Bot.Builder.Tests/Microsoft.Bot.Builder.Tests.csproj

```diff
@@ -1,38 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'm not at all clear why this .csproj file has been changed. Could you explain?

Reply:
We removed this project because it seems _obsolete_. It is an *empty project* where we first thought putting the `TranscriptUtilities` helper. We then moved it to `Microsoft.Bot.Builder.Core.Extensions.Tests`.